metricbeat/module/apache/status/data.go

```diff
@@ -211,40 +211,50 @@ func eventMapping(body io.ReadCloser, hostname string, metricset string) common.
 	}
 
 	event := common.MapStr{
-		"totalAccesses":           totalAccesses,
-		"totalKBytes":             totalKBytes,
-		"uptime":                  uptime,
-		"cpuLoad":                 cpuLoad,
-		"cpuUser":                 cpuUser,
-		"cpuSystem":               cpuSystem,
-		"cpuChildrenUser":         cpuChildrenUser,
-		"cpuChildrenSystem":       cpuChildrenSystem,
-		"reqPerSec":               reqPerSec,
-		"bytesPerSec":             bytesPerSec,
-		"bytesPerReq":             bytesPerReq,
-		"busyWorkers":             busyWorkers,
-		"idleWorkers":             idleWorkers,
-		"connsTotal":              connsTotal,
-		"connsAsyncWriting":       connsAsyncWriting,
-		"connsAsyncKeepAlive":     connsAsyncKeepAlive,
-		"connsAsyncClosing":       connsAsyncClosing,
-		"serverUptimeSeconds":     serverUptimeSeconds,
-		"load1":                   load1,
-		"load5":                   load5,
-		"load15":                  load15,
-		"scbStartingUp":           totalS,
-		"scbReadingRequest":       totalR,
-		"scbSendingReply":         totalW,
-		"scbKeepalive":            totalK,
-		"scbDnsLookup":            totalD,
-		"scbClosingConnection":    totalC,
-		"scbLogging":              totalL,
-		"scbGracefullyFinishing":  totalG,
-		"scbIdleCleanup":          totalI,
-		"scbOpenSlot":             totalDot,
-		"scbWaitingForConnection": totalUnderscore,
-		"scbTotal":                totalAll,
-		"hostname":                hostname,
+		"hostname":      hostname,
+		"totalAccesses": totalAccesses,
+		"totalKBytes":   totalKBytes,
+		"reqPerSec":     reqPerSec,
+		"bytesPerSec":   bytesPerSec,
+		"bytesPerReq":   bytesPerReq,
+		"busyWorkers":   busyWorkers,
+		"idleWorkers":   idleWorkers,
+		"uptime": common.MapStr{
+			"serverUptimeSeconds": serverUptimeSeconds,
```

Comment:
In Apache HTTPD there are 2 main options available. "Standard" and "Extended" status. 
By default, extended status is NOT enabled. It must be configured on purpose.

Fields can be relatively easy separated to both groups.
Standard status: 
- all fields which are not grouped in nested objects 
- Uptime + cpuLoad
- all scoreboard fields (but occurrence is calculated by metrics)
Etended fields:
- the rest of the fields

However, we have some fields not implemented yet. Not even in Apachebeat. They are special functionality fields. For example when SSL is enabled or File cache is turned on. I planed to implement them later since they are not important now. And I also have to test where they will appear when functionality enabled. I believe that they will come to extended status